runtime/appruntime/apisdk/api/authmarshalling/marshal.go

```diff
@@ -0,0 +1,40 @@
+package authmarshalling
+
+import (
+	jsoniter "github.com/json-iterator/go"
+)
+
+var jsonInstance jsoniter.API
```

Comment:
As discussed let's use standard JSON instead, and round-trip auth data via JSON instead